dataforge-io/src/jvmTest/kotlin/hep/dataforge/io/BinaryTest.kt

```diff
@@ -0,0 +1,82 @@
+package hep.dataforge.io
+
+import hep.dataforge.context.Global
+import kotlinx.io.core.readAvailable
+import org.junit.Test
+import java.nio.file.Files
+import java.nio.file.Paths
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class BinaryTest {
```

Comment:
File binary test already exists in https://github.com/mipt-npm/dataforge-core/blob/dev/dataforge-io/src/jvmTest/kotlin/hep/dataforge/io/FileBinaryTest.kt